Fix minor code style issues

dgreen-arm opened the pull request:
## Description
This PR fixes some minor code style issues, such as trailing whitespaces, tabs, etc. TODO's have been removed from two files, with their contents recreated as issues #1443  and #1444 .


## Status
**READY**

## Requires Backporting
NO

Patater commented on library/ssl_tls.c:
```diff
@@ -6893,7 +6893,6 @@ int mbedtls_ssl_read( mbedtls_ssl_context *ssl, unsigned char *buf, size_t len )
     /*
-     * TODO
      *
      * The logic should be streamlined here:
```
Should we delete this whole comment? We copied the comment into #1444 already.

Patater replied:
OK

Patater commented on library/asn1write.c:
```diff
@@ -232,10 +232,6 @@ int mbedtls_asn1_write_int( unsigned char **p, unsigned char *start, int val )
-    // TODO negative values and values larger than 128
-    // DER format assumes 2s complement for numbers, so the leftmost bit
-    // should be 0 for positive numbers and 1 for negative numbers.
-    //
```
This turned into #1443.

Patater replied:
OK

Patater replied:
OK

Patater commented on yotta/data/example-authcrypt/README.md:
```diff
@@ -38,11 +38,11 @@ To build and run this example you must have:
 6. Start the serial terminal emulator and connect to the virtual serial port presented by FRDM-K64F. 
 
-	Use the following settings:
+  Use the following settings:
```
This doesn't render correctly anymore, as per https://github.com/dgreen-arm/mbedtls/blob/849ca7549c49026bb634fde533d0ccd364728662/yotta/data/example-authcrypt/README.md

We need to indent with some more spaces so that Markdown renders this as part of the list, not outside the list item.

Patater replied:
OK

Patater commented on yotta/data/example-benchmark/README.md:
```diff
@@ -38,11 +38,11 @@ To build and run this example you must have:
-	* No flow control. 
+  * 115200 baud (not 9600).
+  * 8N1.
+  * No flow control. 
```
Same issue here with not indenting enough.

Patater replied:
OK

Patater commented on yotta/data/example-hashing/README.md:
```diff
@@ -38,11 +38,11 @@ To build and run this example you must have:
-	* No flow control. 
+  * 115200 baud (not 9600).
+  * 8N1.
+  * No flow control. 
```
Indent more to make this render properly.

Patater replied:
OK

Patater commented on yotta/data/example-selftest/README.md:
```diff
@@ -38,11 +38,11 @@ To build and run this example you must have:
-	* No flow control. 
+  * 115200 baud (not 9600).
+  * 8N1.
+  * No flow control. 
```
Indent more here, otherwise the list isn't rendered correctly.

Patater replied:
OK

mpg commented:
retest

mpg commented:
@Patater github seem to think all your comments are outdated, except one that you note is resolved elsewhere, can you check if you're now happy with the result?
@sbutcher-arm a commit was added since your last review, can you check if you're still happy?

sbutcher-arm commented:
Removing @hanno-arm as reviewer as @Patater and myself should be enough.

Patater commented:
My previous comments were addressed to my satisfaction.

Patater commented:
To ensure we fix all new code that's landed in development since you first made this PR, this needs rebasing on the latest development. A rebase will be easier to review than a merge.

dgreen-arm commented:
I've rebased the PR onto the latest development and added fixes for new code, just some trailing whitespaces. I've also fixed a typo that I found in my rewriting of check-names.sh.

Patater commented:
Only FreeBSD timing test is failing. This is a known flaky test.

sbutcher-arm commented:
@dgreen-arm / @Patater - How are we integrating this into the CI? Do equivalent fixes need to be made on maintained branches?
